### DISCUSSION TITLE
DagsterInvalidSubsetError on Dagster 1.6.12

### DISCUSSION THREAD

**Discussion by johannkm** [dagster-io/dagster#20653] [Q&A]
After upgrading to 1.6.12 I'm hitting

`DagsterInvalidSubsetError(
dagster._core.errors.DagsterInvalidSubsetError: When building job, the AssetsDefinition '<foobar>' contains asset keys [AssetKey()... ] and check keys [], but attempted to select only assets [...] and checks []. This AssetsDefinition does not support subsetting. Please select all asset keys produced by this asset.`

and my code location is failing to load

**Comment by johannkm** [comment]
The temporary solution to this is to pin to dagster 1.6.11. We'll be releasing a fix for this soon.

**Comment by garethbrickman** [comment]
The fix for this has been released [in Dagster Version 1.6.13](https://github.com/dagster-io/dagster/releases/tag/1.6.13)
